### ISSUE TITLE
architectures=avr in library.properties seems to cause some issues.

### ISSUE THREAD

**Issue by MrYsLab** [RobTillaart/DHTstable#10]
I was working on my Telemetrix4Esp8266 server today and noticed a few things you may be interested in.

1. I installed Telemetrix4Arduino using the library manager in a clean Arduino environment. DHTstable is in the depends section of my library.properties file. DHTstable is loaded properly, and I see it where I expect it to be when I click on File/Examples in the Arduino IDE.  I then install Telemetrix4Esp8266, which also specified DHTstable in its depends section, and now DHTstable is moved to INCOMPATIBLE. This happens whether or not I have an avr board or esp8266 board selected in the IDE.

2. Also, as a result of the architectures=avr (I think), when I compile for the ESP8266, I get the following warning:
```
WARNING: library DHTStable claims to run on avr architecture(s) and may be incompatible with your current board, which runs on esp8266 architecture(s).
```

3. Despite the warning, when I run my code against a DHT11 and DHT22 connected to an ESP8266, the library works perfectly.

4. BTW, I tried running DHTNew with my library and the built-in Arduino examples, and both the humidity and temperature report incorrect values as opposed to DHTstable.

**Comment by RobTillaart** [comment]
Great find, i am going to change this architecture flag.
Arduino lint did not complain about this.
It might be also occuring in other libraries, need to check them if they are really only avr.

**Comment by RobTillaart** [comment]
fixed in 1.0.1
